article_store/app.cfg

```diff
@@ -0,0 +1,19 @@
+[django]
+ci = false
+debug = true
+allowed_hosts = web,article-store,localhost,127.0.0.1
+secret_key = FAKE-KEY-DO-NOT-USE-IN-PRODUCTION
+
+[database]
+name: default
+engine: django.db.backends.postgresql
+;host = localhost
+host = postgres
```

Comment:
This seems to be the actual configuration file (pointing to `postgres:5432`) therefore should not be in the image, but mounted from the outside as a volume by docker-compose.

The alternative would be to pass in environment variables, but the values here seem involved enough that we don't want to define one for each.